dev/e2e_app/integration_test/camera_features/pick_multiple_image_from_gallery_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+import '../common.dart';
+import 'camera_helpers.dart';
+
+void main() {
+  patrol('pick multiple images from gallery', ($) async {
+    await createApp($);
+    final cameraHelpers = CameraHelpers($);
+    // debugPrint('getNativeViews2: ${await $.native2.getNativeViews(null)}');
+    debugPrint('getNativeViews: ${await $.native.getNativeViews(null)}');
+    if (await $.native.isVirtualDevice() && Platform.isAndroid) {
```

Comment:
Could this `if` be inside a method which would merge both real device and emulator methods?